readme.md

```diff
@@ -45,50 +48,16 @@ Libraries are bundled into the point of use, therefore if you try transpile a st
-| :---------------------------------: | :---------------: |
-|   Will likely never be supported    |        :x:        |
-|    Being developed/investigated     | :hammer_and_pick: |
-| Currently Unknown/If added in Cairo |    :question:     |
-
-|                      Solidity                       |  Support Status   |
-| :-------------------------------------------------: | :---------------: |
-|            fallback functions with args             | :hammer_and_pick: |
-|                   delegate calls                    | :hammer_and_pick: |
-|               indexed arrays in event               | :hammer_and_pick: |
-|                   low level calls                   |        :x:        |
-|              nested tuple expressions               |    :question:     |
-|                      gasleft()                      |    :question:     |
-|                      msg.value                      |    :question:     |
-|                       msg.sig                       |    :question:     |
-|                      msg.data                       |    :question:     |
-|                     tx.gasprice                     |    :question:     |
-|                      tx.origin                      |    :question:     |
-|                      try/catch                      |    :question:     |
-|                   block.coinbase                    |    :question:     |
-|                   block.gaslimit                    |    :question:     |
-|                    block.basefee                    |    :question:     |
-|                    block.chainid                    |    :question:     |
-|                  block.difficulty                   |        :x:        |
-|         precompiles (apart from ecrecover)          |    :question:     |
-|                    selfdestruct                     |    :question:     |
-|                      blockhash                      |    :question:     |
-|            functions pointers in storage            |    :question:     |
-|           sha256 (use keccak256 instead)            |        :x:        |
-|                       receive                       |    :question:     |
-|  Inline Yul Assembly - (memory, calldata, storage)  |    :question:     |
-|                 user defined errors                 |    :question:     |
-|   function call options e.g x.f{gas: 10000}(arg1)   |    :question:     |
-| member access of address object e.g address.balance |    :question:     |
-|              nested tuple assignments               |    :question:     |
-
-Note: We have changed the return of `ecrecover` to be `uint160` because we use the `address` type for Starknet addresses.
+**Solidity**: Warp supports Solidity 0.8 and 0.7. In order to support newer versions [nethersolc](https://github.com/NethermindEth/nethersolc) has to be updated, and [nethersolc binaries](https://github.com/NethermindEth/warp/tree/develop/nethersolc) within warp repository have to be updated.
```

Comment:
Warp only supports 0.8.
We haven't test 0.7 fully and we should not say we support it